src/systems/abstractsystem.jl

```diff
@@ -870,6 +870,11 @@ function namespace_guesses(sys)
     Dict(unknowns(sys, k) => namespace_expr(v, sys) for (k, v) in guess)
 end
 
+function namespace_parameter_dependencies(sys)
+    pdeps = get_parameter_dependencies(sys)
```

Comment:
```suggestion
    pdeps = parameter_dependencies(sys)
```

This needs to be recursive, since we can have multiple levels of nesting